egs/libri_css/s5_mono/local/download_diarizer.sh

```diff
@@ -35,4 +35,4 @@ rm 0012_diarization_v1/exp/xvector_nnet_1a/plda
 wget https://desh2608.github.io/static/files/jsalt/plda -P 0012_diarization_v1/exp/xvector_nnet_1a/
 cd ../..
 cp -r ${dir}/0012_diarization_v1/exp .
-rm -rf ${dir}
+trap "rm -rf ${dir}" EXIT
```

Comment:
Move the trap line right after the directory is created at line 24. Here, as the last line in the file, it's same as `rm -rf ${dir}`. The idea of `trap` is to schedule a command to be run when an event (`EXIT` in this case) happens.